pmlc/dialect/pxa/ir/ops.td

```diff
@@ -56,26 +64,26 @@ def PxaReduceOp : PxaReduceOpBase<"reduce"> {
     AffineMapAttr:$map,
     Variadic<Index>:$idxs
   );
-  let results = (outs AnyMemRef:$result);
+  let results = (outs AnyMemRef : $result);
 
   let hasCanonicalizer = 1;
   let hasFolder = 1;
 
-  let builders = [
-    OpBuilder<(ins
-      "mlir::AtomicRMWKind":$agg,
-      "mlir::Value":$val,
-      "mlir::Value":$mem,
-      "mlir::AffineMap":$map,
-      "mlir::ValueRange":$idxs
-    ), [{
-      build($_builder, $_state, mem.getType(), agg, val, mem, map, idxs);
-    }]>
-  ];
+  let builders = [OpBuilder<(ins "mlir::AtomicRMWKind"
+                             : $agg, "mlir::Value"
+                             : $val, "mlir::Value"
+                             : $mem, "mlir::AffineMap"
+                             : $map, "mlir::ValueRange"
+                             : $idxs),
+                            [{
```

Comment:
The original style matches closer to upstream. I think we should try to match upstream style as much as possible to make submission to the pxa dialect to MLIR core easier.